src/main/java/gameframework/motion/GameMovable.java

```diff
@@ -40,7 +51,10 @@ public void setDriver(GameMovableDriver driver) {
 	public GameMovableDriver getDriver() {
 		return moveDriver;
 	}
-
+	
+	/**
+	 * Realize a single movement based on the Speed and the Direction of the SpeedVector
```

Comment:
English: /realize/ is devoid of meaning. Instead of /realize a shift in position/, just say /move/…